get_data.py

```diff
@@ -0,0 +1,108 @@
+"""Module to Store Data From the CSV files into data Structures and Handle the Parsed Commands"""
+
+import os
+import csv
+from daily_weather_data import WeatherDataRow
+
+
+class WeatherData:
+    """Class containing 2 class variables (lists) to use .
+    Contains attributes for file path , year to display and the month for each object
+    Contains a method to load monthly and yearly data
+    Contains an internal function which cuts out useless columns from the Monthly_weather dictionary and yearly_weather dictionaries
+    """
+
+    columns_to_keep = [
+        "PKT",
+        "Max TemperatureC",
+        "Mean TemperatureC",
+        "Min TemperatureC",
+        "Max Humidity",
+        " Mean Humidity",
+    ]
+    months_of_the_year = [
+        "jan",
+        "feb",
+        "mar",
+        "apr",
+        "may",
+        "jun",
+        "jul",
+        "aug",
+        "sep",
+        "oct",
+        "nov",
+        "dec",
+    ]
+
+    def __init__(self, parser_file_path, year, month):
+        """initialize function to store attributes affter instantiation of a class"""
+
+        self.parser_file_path = parser_file_path
+        self.year = year
+        self.month = month
+
+    def load_data(self, month=None):
+        """Load data for a specific month or all months if month is None.
+        Returns a List of Objects if the month is given in the format :
+
+        list = [obj1 , obj2 , obj3]  each obj contains values for Max , Min temp and humidity
+
+        The Yearly Data consists of a 2D list conttaining 12 Lists of Objects ,corresponding to each month
+        with each object corresponding to a single day of the month
+
+        2Dlist_yearly = [[obj1 , obj2 , obj3] , [obj4 , obj5 , obj6]]
+
+
+        """
```

Comment:
Function's return type should be consistent either return [[WeatherDataRow,...],...] or [WeatherDataRow,...]